contracts/utils/CurvedInterestRate.sol

```diff
@@ -85,6 +85,14 @@ contract CurvedInterestRate is PausableGuardian_0_8, ICurvedInterestRate {
     }
 
     function updateParams(CurveIRParams calldata _curveIRParams, address owner) public onlyGuardian {
+        // updateParams((120e18, 80e18, 100e18, 100e18, 110e18, 0.1e18, 0.01e18), ZERO_ADDRESS, {"from": deployer}) # default across all
+        require(_curveIRParams.IR2 <= CHECK_IR2, "IR2");
+        require(_curveIRParams.UR1 <= CHECK_UR_MAX, "UR1");
+        require(_curveIRParams.UR2 <= CHECK_UR_MAX, "UR2");
+        require(_curveIRParams.UR_MAX <= CHECK_UR_MAX, "UR_MAX");
+        require(_curveIRParams.IR_MAX <= CHECK_IR2, "IR_MAX");
+        require(_curveIRParams.IR_MIN >= CHECK_IR_MIN && _curveIRParams.IR_MIN <= CHECK_IR2, "IR_MIN");
+        require(_curveIRParams.IR_ABSOLUTE_MIN >= CHECK_IR_ABSOLUTE_MIN && _curveIRParams.IR_ABSOLUTE_MIN < CHECK_IR_MAX, "IR_ABSOLUTE_MIN");
```

Comment:
absolute min does not need any checks and should be fixed at 1e11 as it only exists to ensure that TWAI does not mis-calculate